raiden/utils/debugging.py

```diff
@@ -79,6 +79,7 @@ class Idle:
     before_poll: float = field(init=False, default_factory=time.time)
     last_print: float = field(init=False, default_factory=time.time)
     measurements: List[IdleMeasurement] = field(init=False, default_factory=list)
+    before: Optional[float] = None
```

Comment:
IMO, this doesn't have to be optional. We can initialize it to `time.time()` and `prepare_handler` will override that value once the hub kicks in.

Reply:
this was a typo, I renamed the value to `before_poll`